t/lib/t/MusicBrainz/Server/Controller/Artist/Ratings.pm

```diff
@@ -56,7 +56,7 @@ test 'Test artists rating page' => sub {
     );
     $tx->is(
         '(//span[@class="current-rating"])[2]',
-        3.5,
+        3,
```

Comment:
This looks like it was just wrong, but maybe someone can confirm my math here.

The SQL at the top adds two raw ratings, 20 and 100.  (20 + 100) / 2 / 20 = 3.  So I'm not even sure where it was getting 3.5 from.

Reply:
The whole issue here seems to be that the controller_artist sql file does:

`UPDATE artist_meta SET rating=70, rating_count=4 WHERE id = 3;`

Since we were not earlier updating the meta tables with triggers, so that was just set separately. You can probably remove that as well, if you haven't (and check for any other tests that update the meta tables).